cms/core/tests/test_views.py

```diff
@@ -17,3 +20,16 @@ def test_csrf_token_mismatch_logs_an_error(self):
             self.client.post("/admin/login/", {})
 
         self.assertIn("CSRF Failure: CSRF cookie", logs.output[0])
+
+
+class TestCookiesBannerTestCase(TestCase):
+    """Test for the cookie banner functionality."""
+
+    def test_google_tag_manager_script_present_when_tracking_consent_given(self):
+        """Check that the Google Tag Manager script is present on the page when the user agrees to tracking."""
+        self.client.cookies = SimpleCookie({"ons_cookie_policy": "'usage':true"})
```

Comment:
suggestion:
```suggestion
        self.client.cookies["ons_cookie_policy"] = "'usage':true"
```

because `self.client.cookies` is already a `SimpleCookie` morsel ([ref](https://docs.djangoproject.com/en/5.1/topics/testing/tools/#django.test.Client.cookies)), which acts like a dict. Doing it this way ensures we don't accidentally overwrite anything else, and it should not break in the future.

Reply:
Good shout, I'll do that and also change the cookie value value to dict rather than a string.